packages/homepage/content/articles/new-dashboard/index.md

```diff
@@ -0,0 +1,93 @@
+---
+banner: ./images/banner.png
+slug: new-dashboard
+authors: ["Ives van Hoorne"]
+photo: https://avatars0.githubusercontent.com/u/587016?s=460&v=4
+title: A New-look Dashboard
+description:
+  Our new dashboard will get you started faster and keep your code organized.
+date: 2020-07-08
+---
+
+## Dashboard Update
+
+The dashboard is a core part of working on CodeSandbox, and it's of increasing
+importance to power users and teams. While staying close to what people are
+familiar with, we've refreshed the design of the dashboard. This update evolves
+the design to bring through the
+[updates made in the editor](https://codesandbox.io/post/new-look-editor) and
+makes working with teams and more sandboxes a better experience. But these
+changes aren't just cosmetic; they add new functionality that gives you greater
```

Comment:
Yeah I like this part a lot. I think we should start with talking about the new introduced concept of workspaces, hopefully this encourages people to try it out too. Maybe we should add a link to creating a new workspace as well.